src/main/java/mate.academy/Main.java

```diff
@@ -10,7 +10,8 @@ public class Main {
     public static void main(String[] args) {
         // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
-        ProductService productService = null;
+        Object instance = injector.getInstance(ProductService.class);
+        ProductService productService = (ProductService) instance;
```

Comment:
You can cast to ProductService in one line
```suggestion
        ProductService productService = (ProductService) injector.getInstance(ProductServiceImpl.class);
```